prow/config/branch_protection.go

```diff
@@ -239,7 +248,11 @@ func (c *Config) GetPolicy(org, repo, branch string, b Branch) (*Policy, error)
 	if prowContexts, _, _ := BranchRequirements(org, repo, branch, c.Presubmits); len(prowContexts) > 0 {
 		// Error if protection is disabled
 		if policy.Protect != nil && !*policy.Protect {
-			return nil, fmt.Errorf("required prow jobs require branch protection")
+			if c.BranchProtection.AllowDisabledJobPolicies {
+				logrus.Warnf("%s/%s=%s has required jobs but has protect: false", org, repo, branch)
+			} else {
+				return nil, fmt.Errorf("required prow jobs require branch protection")
```

Comment:
Can we maybe warn once?

Reply:
Sorry, what do you mean by this?

Reply:
I would like to still like to tell the user they are bad and they should feel bad (aka they really ought to turn on branch protection)

If you think doing that for every unprotected tide repo then maybe just log this message once per session?

Reply:
The usefulness of this current error is that you get one per branch that is misconfigured so if you are to fix it by making them protected you know which need to change. Also to be clear with the current impl of the error this kills the process